From f5edadb42d7d0cf893304d40f9752dfff58a9c52 Mon Sep 17 00:00:00 2001
From: Noam.Lessner <nlessner@habana.ai>
Date: Wed, 01 Sep 2021 14:28:13 +0300
Subject: [PATCH] [SW-0] gaudi2: preboot: arc: decrease stack

decrease stack size to allow more code to fit in sram region.

Change-Id: If3107d9cf44014c4607473125206ea3fa6e5ccab
(cherry picked from commit a845e28d15774618e4871a972fe8d9ed50a69664)
---

diff --git a/arch/gaudi2/arc/preboot.lds.S b/arch/gaudi2/arc/preboot.lds.S
index 1bc858f..a467ebd 100644
--- a/arch/gaudi2/arc/preboot.lds.S
+++ b/arch/gaudi2/arc/preboot.lds.S
@@ -31,7 +31,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 4096 ;
+		. = . + 2048 ;
 		__stack_top = . ;
 	} > .sram
 
